x-pack/plugins/cases/public/components/connectors/card.tsx

```diff
@@ -20,12 +19,6 @@ interface ConnectorCardProps {
   isLoading: boolean;
 }
 
-const StyledText = styled.span`
```

Comment:
Removed all styles. I converted the component to use `EuiFlex*` instead of `EuiCard`. The result is the same as before.